aperturedb/Connector.py

```diff
@@ -55,7 +62,8 @@ class Session():
     def valid(self) -> bool:
         session_age = time.time() - self.session_started
 
-        if session_age > self.session_token_ttl:
+        # This triggers refresh if the session is about to expire.
+        if session_age > self.session_token_ttl - 10:
```

Comment:
can we make the 10 a variable we define at the top like `SESSION_EXPIRTY_OFFSET_SEC` ?